tech/languages/csharp/dotnet-installation.md

```diff
@@ -19,16 +19,19 @@ $ sudo dnf copr enable @dotnet-sig/dotnet
 
 .NET Core SDK will enable building and publishing C# source code.
 ```
-$ sudo dnf install dotnet-sdk-2.0
+$ sudo dnf install dotnet-sdk-2.1
```

Comment:
Is there any way to make this generic (install latest)?
e.g.: `$ sudo dnf install dotnet-sdk`

Reply:
This is outdated, as there was other PR in the meantime.